### PR TITLE
Add set of APIs to set/get the global state

### PR DESCRIPTION
**Pull request by 0xjei**
This PR adds two endpoints to the server to set and retrieve the last message (aka color) present in the last PCD consumed by the client. The color is saved in an environment variable on NodeJS to keep things minimal (no db).

__TODOs__
- [x] Add get / post endpoints on server
- [x] Add missing middlewares (json + cors)
- [x] Adapt client logic

### CODE REVIEW

File: apps/client/src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+    useEffect(() => {
+        // Update the background color accordingly to color change.
+        const appElement = document.getElementById("app")!
+
+        appElement.style.backgroundColor = `#${bgColor}`
+    }, [bgColor])
+
+    useEffect(() => {
+        ; (async () => {
+            // @ts-ignore
```
Comment (cedoor) [inline apps/client/src/App.tsx:41]: Why is this necessary?

File: apps/client/src/App.tsx [hunk 2]
```diff
@@ -36,13 +74,13 @@ export default function App() {
 
     const getEdDSAPCD = useCallback(() => {
         const url = getWithoutProvingUrl(
-            process.env.PCDPASS_URL,
+            String(process.env.PCDPASS_URL),
```
Comment (cedoor) [inline apps/client/src/App.tsx]: This shouldn't need a conversion, it should already be a string right?

Comment (0xjei) [reply apps/client/src/App.tsx]: Amend in the next commit

File: apps/client/src/App.tsx (hunk 1 above)
Comment (0xjei) [reply apps/client/src/App.tsx:41]: Potential `undefined` on `EdDSAPCDPackage` from `@pcd/eddsa-pcd` package. Need further investigation, ignored for now.

**Review by cedoor — APPROVED** [review]
